### ISSUE TITLE
GitHub action `version-update.yml` is failing

### ISSUE THREAD

**Issue by hirasso** [swup/cli#38]
**Description of the issue**

I can't seem to run the [Update package version](https://github.com/swup/cli/actions/runs/6316240898) action, even though it's identical to the equivalent in e.g. fragment plugin where it worked yesterday. Any ideas @daun?

**How to reproduce the issue**

Run the action manually > it fails

**Comment by daun** [comment]
I think the npm scripts might require additional `npx` prepended?

**Comment by hirasso** [comment]
Once again, no idea 😁

**Comment by hirasso** [comment]
Is this really related to the npm scripts? They only run in the publish action, don't they?

**Comment by daun** [comment]
In this case, there's a `version` script that npm seems to try and run:

```sh
> version
> oclif readme && git add README.md

oclif: not found
```

**Comment by daun** [comment]
@hirasso I guess your PR earlier solved this issue, right?

**Comment by hirasso** [comment]
Ah yes! Solved in #47
